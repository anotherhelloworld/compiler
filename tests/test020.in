a, b, c: integer;
a:=10;
a+=c;
b: boolean;