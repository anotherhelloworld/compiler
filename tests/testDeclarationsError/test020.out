Expression ":" expected but "=" found in Line 2 Column 8
